pkgs/development/python-modules/nose-of-yeti/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, six, nose, fudge, should-dsl }:
+
+buildPythonPackage rec {
+  pname = "noseOfYeti";
```

Comment:
```suggestion
  pname = "nose-of-yeti";
```

Reply:
Thanks, fixed now! I just used the "official" PyPI name here as I thought it should be used here, although I used the normalised name elsewhere.